### PR TITLE
Fix PushPullIT errors

### PR DESCRIPTION
**Pull request by davidxia**

### CODE REVIEW

File: src/test/java/com/spotify/docker/it/PushPullIT.java [hunk 1]
```diff
@@ -138,7 +138,7 @@ public void testPushImageToPrivateAuthedRegistryWithoutAuth() throws Exception {
     client.build(Paths.get(dockerDirectory), LOCAL_IMAGE);
 
     exception.expect(ImagePushFailedException.class);
-    exception.expectMessage("no basic auth credentials");
+    exception.expectMessage("Authentication is required");
```
Comment (davidxia) [inline src/test/java/com/spotify/docker/it/PushPullIT.java]: The registry seems to have been updated so this string changed: https://hub.docker.com/r/library/registry/tags/. If it changes again, I'll look into fixing the version we pull.

**Comment by davidxia** [comment]
@mattnworb @negz

**Comment by mattnworb** [comment]
Does this fix all the integration tests? If so might as well change the travis script to `verify` instead of `mvn test`

**Comment by davidxia** [comment]
I'm doing some more local testing just to make sure, but yes, that's the
goal.

On Fri, Apr 1, 2016 at 9:48 AM, Matt Brown <notifications@github.com> wrote:

> Does this fix all the integration tests? If so might as well change the
> travis script to verify instead of mvn test
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/docker-client/pull/409#issuecomment-204402221>
>



-- 
David Xia
Software Engineer
dxia@spotify.com

**Comment by davidxia** [comment]
This should be good to go now.

**Pull request merged**